test/handler/test_fsc_executable_and_command.vader

```diff
@@ -0,0 +1,18 @@
+Before:
+  runtime ale_linters/scala/fsc.vim
+
+After:
+  call ale#linter#Reset()
+
+Given scala(An empty Scala file):
+
+Execute(The default executable and command should be correct):
+  AssertEqual 'fsc', ale_linters#scala#fsc#GetExecutable(bufnr(''))
+  AssertEqual
+  \ ale#Escape('fsc') . ' -Ystop-after:parser %t',
+  \ ale_linters#scala#fsc#GetCommand(bufnr(''))
+
+Given scala.sbt(An empty SBT file):
+Execute(fsc should not be run for sbt files):
+  AssertEqual '', ale_linters#scala#fsc#GetExecutable(bufnr(''))
+  AssertEqual '', ale_linters#scala#fsc#GetCommand(bufnr(''))
```

Comment:
You can move the tests for `GetExecutable` or `GetCommand` to `test/command_callback`.